fixture-monkey-junit-jupiter/src/main/java/com/navercorp/fixturemonkey/junit/jupiter/extension/FixtureMonkeySeedExtension.java

```diff
@@ -0,0 +1,31 @@
+package com.navercorp.fixturemonkey.junit.jupiter.extension;
+
+import org.junit.jupiter.api.extension.AfterTestExecutionCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.lang.reflect.Method;
+import java.util.logging.Logger;
+
+public class FixtureMonkeySeedExtension implements AfterTestExecutionCallback {
+
+	private static final Logger logger = Logger.getLogger(FixtureMonkeySeedExtension.class.getName());
+
+	private static final ThreadLocal<Long> seedHolder = new ThreadLocal<>();
+
+	/**
+	 * Store the seed used in the Fixture Monkey builder before test execution and output that seed if the test fails
+	 * @param seed the seed used in the Fixture Monkey builder
+	 */
+	public static void setSeed(long seed) {
+		seedHolder.set(seed);
+	}
```

Comment:
I think it is better to use the seed in `Randoms`.

You can get it with `currentSeed` it is also used by the `seed` option in FixtureMonkeyBuilder.